src/service.ts

```diff
@@ -234,7 +234,25 @@ export class DebuggerService implements IDebugger, IDisposable {
         );
       });
     }
-
+    const unassociatedBreakpoints = (
```

Comment:
Don't we want to keep the change related to the breakpoints in a separate PR, and keep this PR focused on the refactor of the tracker? (extracting the "find in" logic into a separate plugin)

What do you think?

Reply:
It makes sense. I will remove this method from this PR.

Reply:
I've just removed.